packages/brick-container/src/index.tsx

```diff
@@ -63,7 +63,13 @@ const api = `${getRuntime().getBasePath()}api/gateway/data_exchange.store.ClickH
 const analyzer = apiAnalyzer.create({
   api,
 });
-
+http.interceptors.request.use(function (config: HttpRequestConfig) {
+  config.options = {
+    ...config.options,
+    headers: new Headers({ lang: i18n.resolvedLanguage }),
+  };
+  return config;
```

Comment:
首先，注意 `options` 里可能已经有其它的 headers 的；其次，最好不要 mutate 原数据：

```suggestion
  const headers = new Headers(config.options?.headers || {});
  headers.set("lang", i18n.resolvedLanguage);
  return {
    ...config,
    options: {
      ...config.options,
      headers,
    },
  };
```

Reply:
好的，已经修改，以后一定注意。